MobiFlight/MobiFlightFirmwareUpdater.cs

```diff
@@ -89,9 +89,6 @@ public static bool UpdateFirmware(MobiFlightModule module, String FirmwareName)
 
         public static void RunAvrDude(String Port, Board board, String FirmwareName) 
         {
-            String ArduinoChip = board.AvrDudeSettings.Device;
-            String Bytes = board.AvrDudeSettings.BaudRate;
-            String C = board.AvrDudeSettings.Programmer;

```

Comment:
These three lines should have been removed back when I did the .board.json work. They aren't needed and the properies are now just referenced directly in the `anyCommand` string construction.